packages/common/src/store/ui/buy-sell/useBuySellSwap.ts

```diff
@@ -39,6 +48,25 @@ export const useBuySellSwap = (props: UseBuySellSwapProps) => {
   const retryTimeoutRef = useRef<NodeJS.Timeout | null>(null)
   const lastSwapDataRef = useRef<any>(null)
 
+  const { data: baseCoin } = useArtistCoin(selectedPair.baseToken.address ?? '')
```

Comment:
should these be disabled on the address to prevent unnecessary reqs? not sure if necessary but

Reply:
the hook has that built in